mandatory/4-tax.js

```diff
@@ -17,7 +20,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(product) {
+  let amountOne=( (product*1.2).toFixed(2)).toString();
```

Comment:
Hello @Jacknguyen4438 ,
Can you think of a better variable name than `amountOne`? Think about the calculation you're doing here and what it means in order to help you come up with a clearer variable name. Remember your variable names should convey meaning, as your code will be read by other humans.